pandas/core/series.py

```diff
@@ -1096,8 +1096,7 @@ def iteritems(self):
         """
         return zip(iter(self.index), iter(self))
 
-    if compat.PY3:  # pragma: no cover
-        items = iteritems
```

Comment:
I think `compa.iteritems` just calls the object's `iteritems` method, so that would not make a difference?